packages/migrate/src/__tests__/MigrateDiff.test.ts

```diff
@@ -18,42 +18,14 @@ const describeIf = (condition: boolean) => (condition ? describe : describe.skip
 const originalEnv = { ...process.env }
 
 describe('migrate diff', () => {
-  describe('generic', () => {
-    it('wrong flag', async () => {
-      const commandInstance = MigrateDiff.new()
-      const spy = jest.spyOn(commandInstance, 'help').mockImplementation(() => 'Help Me')
-
-      await commandInstance.parse(['--something'])
-      expect(spy).toHaveBeenCalledTimes(1)
-      spy.mockRestore()
-    })
-
-    it('help flag', async () => {
-      const commandInstance = MigrateDiff.new()
-      const spy = jest.spyOn(commandInstance, 'help').mockImplementation(() => 'Help Me')
-
-      await commandInstance.parse(['--help'])
-      expect(spy).toHaveBeenCalledTimes(1)
-      spy.mockRestore()
-    })
-
+  describe('D1', () => {
     it('should succeed when --from-local-d1 and a single local Cloudflare D1 database exists', async () => {
       ctx.fixture('cloudflare-d1-one-db')
 
       const result = await MigrateDiff.new().parse(['--to-empty', '--from-local-d1', '--script'])
       expect(result).toMatchInlineSnapshot(``)
       expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
-      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
-        -- DropTable
-        PRAGMA foreign_keys=off;
-        DROP TABLE "Post";
-        PRAGMA foreign_keys=on;
-
-        -- DropTable
-        PRAGMA foreign_keys=off;
-        DROP TABLE "User";
-        PRAGMA foreign_keys=on;
-      `)
+      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
```

Comment:
Please replace `ctx.mocked['console.info']` with `ctx.mocked['process.stdout.write']`, so we don't lose track of the SQL script in the snapshot.